fvm/evm/evm_test.go

```diff
@@ -1770,7 +1770,7 @@ func TestDryRun(t *testing.T) {
 				require.Less(t, result.GasConsumed, limit)
 
 				// gas limit too low, but still bigger than intrinsic gas value
-				limit = uint64(21216)
+				limit = uint64(24_216)
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .